Task8/.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
+    "plugins": [
+        "react"
+    ],
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "airbnb",
+        "airbnb/hooks" 
+    ],
+    "rules": {
+        "no-set-state": "off",
+        "react/destructuring-assignment": 1,
+        "react/prop-types": 1,
+        "prefer-promise-reject-errors" : 1,
+        "no-undef" : 1,
+        "linebreak-style": ["error", "windows"],
+        "react/jsx-indent": ["error", 4],
+        "react/jsx-indent-props": ["error", 4],        
+        "indent": ["error", 4,  {"SwitchCase": 1}],
```

Comment:
Please, update `eslint` package to `6.6.0` and then fix indention (it seems that bug which we noticed last week has been fixed in new version of `eslint` package).

Reply:
And also fix other `eslint` errors throughout all project.

Reply:
Fixed